requirements/customer/product-environment.tex

```diff
@@ -0,0 +1,46 @@
+\section{Product Environment}
+  \subsection{User Client}
+    \subsubsection{Hardware Requirements}
+      \begin{itemize}
+      \item A connection to a LAN with a central server.
```

Comment:
Maybe just `An internet connection to the central server`? I don't think we need it to be LAN (can be WLAN for ex.), although the terms are used often used interchangeably.